application/__init__.py

```diff
@@ -17,13 +17,24 @@
 #   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 #   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 #   IN THE SOFTWARE.
-
+import logging.config
+import yaml
 import os
 import sys
 
 from flask import Flask, Blueprint
 from werkzeug.serving import WSGIRequestHandler
```

Comment:
`yaml` is a 3rd party (non-Python) import and shall be organized into another group. Sorry for missing this in the last review.
```suggestion
import logging.config
import os
import sys

import yaml
from flask import Flask, Blueprint
from werkzeug.serving import WSGIRequestHandler
```